README-es.md

```diff
@@ -0,0 +1,286 @@
+<p align="center">
+    <img alt="oss image" src="./imgs/zoss-logo.svg" height="200px" width="200px">
+</p>
+
+<p align="center">
+  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">
+    <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />
+  </a>
+</p>
+
+<h1 align="center">Hoja de referencia de promoción al código abierto</h1>
+
+
+> Esta hoja de referencia resume los pasos importantes que debes seguir para promover tu proyecto de código abierto en óptimas condiciones. Puedes hacer clic en los elementos para expandir y obtener más información de un tema.
+
+Lista de lenguajes disponibles:
+
+- [Inglés](./README.md)
```

Comment:
No need to translate these countries in Spanish (except Spanish) 😉

Reply:
Okok